src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/AspNetAppElasticBeanstalkLinux.csproj

```diff
@@ -25,9 +25,8 @@
 
   <ItemGroup>
     <!-- CDK Construct Library dependencies -->
-    <PackageReference Include="Amazon.CDK" Version="1.128.0" />
-    <PackageReference Include="Amazon.CDK.AWS.ElasticBeanstalk" Version="1.128.0" />
-    <PackageReference Include="Amazon.CDK.AWS.S3.Assets" Version="1.128.0" />
+    <PackageReference Include="Amazon.CDK.Lib" Version="2.13.0" />
+    <PackageReference Include="Constructs" Version="10.0.70" />
```

Comment:
Since `Amazon.CDK.Lib` has a dependency on `Constructs` you don't need to add that `PackageReference`.

Reply:
Done